skimage/measure/__init__.py

```diff
@@ -7,7 +7,7 @@
 from ._pnpoly import points_in_poly, grid_points_in_poly
 from ._moments import moments, moments_central, moments_normalized, moments_hu
 from .profile import profile_line
-from .fit import LineModel, CircleModel, EllipseModel, ransac
+from .fit import LineModel, LineModel3D, CircleModel, EllipseModel, ransac
```

Comment:
I agree, but it seems low priority as all other models are also 2D (although circle turn to spheres, ellipses to ellipsoids while line2Ds turn to line3Ds). Let me know if you would like me to do it, and then I would appreciate if you could point me to an example of "code deprecation".